rules/S6636/cfamily/metadata.json

```
@@ -0,0 +1,17 @@
+{
+  "title": "\"unhandled_exception\" should not be empty",
+  "type": "BUG",
+  "status": "ready",
+  "remediation": {
+    "func": "Constant\/Issue",
+    "constantCost": "10min"
+  },
+  "tags": [
+  ],
+  "defaultSeverity": "Blocker",
+  "ruleSpecification": "RSPEC-6636",
+  "sqKey": "S6636",
+  "scope": "All",
+  "defaultQualityProfiles": ["Sonar way"],
+  "quickfix": "unknown"
```

```suggestion
  "quickfix": "infeasible"
```
I don't know how the user handles exceptions, so it is unlikely you'll be able to generate this function automatically